access.go

```diff
@@ -599,14 +599,6 @@ func (s *Server) FinishAccessRequest(w *Response, r *http.Request, ar *AccessReq
 			w.Storage.RemoveAuthorize(ret.AuthorizeData.Code)
 		}
 
-		// remove previous access token
-		if ret.AccessData != nil && !s.Config.RetainTokenAfterRefresh {
-			if ret.AccessData.RefreshToken != "" {
-				w.Storage.RemoveRefresh(ret.AccessData.RefreshToken)
-			}
-			w.Storage.RemoveAccess(ret.AccessData.AccessToken)
```

Comment:
we still want to revoke the access token

Reply:
I have fixed it in this commit be9c164